governance/tsc-procedures.md

```diff
@@ -0,0 +1,16 @@
+# Technical Steering Committee (TSC) Procedures
+## TSC Membership
+Membership changes to the Paraglider TSC is governed by the current list of TSC members with voting rights. TSC membership changes require a majority vote by the current TSC members in order to pass. For more details on TSC voting procedures please refer to the Paraglider Technical Charter "TSC Voting" section.
+
+## Adding TSC Members
+In order to have a vote on adding a new member to the Paraglider TSC a github issue should be opened against the repository and tagged with the “governance” label. The title of the issue should be TSC New Member - <NAME> with a description on why this person should be added to the TSC. The opened github issue should then be referenced in the next TSC meeting which will determine a date to conduct a vote on the new membership. If a vote does not occur within a 1-month period from opening the issue the motion will be considered denied, and the issue will be closed.
```

Comment:
Assuming you want \<NAME> to be visible, I think you need to escape at least the opening angle bracket in markdown.

```suggestion
In order to have a vote on adding a new member to the Paraglider TSC a github issue should be opened against the repository and tagged with the “governance” label. The title of the issue should be TSC New Member - \<NAME> with a description on why this person should be added to the TSC. The opened github issue should then be referenced in the next TSC meeting which will determine a date to conduct a vote on the new membership. If a vote does not occur within a 1-month period from opening the issue the motion will be considered denied, and the issue will be closed.
```

Reply:
Also, an example of how I would format long paragraphs to optimize for changeability and diffing:

```suggestion
In order to have a vote on adding a new member to the Paraglider TSC a github issue should be opened against the repository and tagged with the “governance” label.
The title of the issue should be TSC New Member - \<NAME> with a description on why this person should be added to the TSC.
The opened GitHub issue should then be referenced in the next TSC meeting which will determine a date to conduct a vote on the new membership.
If a vote does not occur within a 1-month period from opening the issue the motion will be considered denied, and the issue will be closed.
```